packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -101,6 +111,7 @@ function MainLayoutHeader({
           {!hideButton && user && (
             <LinkWithTooltip
               tooltip={{ placement: 'left', content: 'Notifications' }}
+              onClick={onNavigateNotifications}
```

Comment:
Shouldn't we add the same callback to the mobile button as well?

Reply:
Yes, it should be the line below, from `FooterNavBar`:
https://github.com/dailydotdev/apps/blob/a50edc87ec402b47b60a2ac8f51efdd044cf7669/packages/webapp/components/FooterNavBar.tsx#L93-L103